lob/api_requestor.py

```diff
@@ -55,13 +56,24 @@ def request(self, method, url, params=None):
             del params['headers']
 
         if method == 'get':
-            return self.parse_response(
-                requests.get(lob.api_base + url, auth=(self.api_key, ''), params=params, headers=headers)
-            )
+            try:
+                resp = requests.get(
+                    lob.api_base + url, 
+                    auth=(self.api_key, ''),
+                    params=params,
+                    headers=headers,
+                    timeout=timeout
+                    )
+                return self.parse_response(resp)
+            except requests.exceptions.Timeout:
```

Comment:
I don't think we should be catching and re-raising a custom `Timeout` exception.
Just let the `request Timeout` bubble up.

Reply:
I was having trouble creating a test for this. The `requests_mock` package doesn't trigger timeouts when you do a slowed down response using `time.sleep`. You can raise a `requests.exception` with the mock, but our library doesn't really take any action based on that exception. I figured since we are using timeout as one of the parameters that we are passing in now, we could have our custom exception for our library. And its easier to test, since our library does some action based on the exception.

Reply:
Shouldn't need to use `time.sleep`, `requests-mock` has a dedicated method of simulating timeouts.
https://requests-mock.readthedocs.io/en/latest/response.html#raising-exceptions

Reply:
@Kilo59 I'm gonna try a different approach and see if I intercept the http request and supply my own response function. If that doesn't work, I'll see if I can put spy to make sure that we are passing down the timeout value.